client/settings/fraud-protection/components/protection-levels/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import HelpOutlineIcon from 'gridicons/dist/help-outline';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useCurrentProtectionLevel, useCurrencies } from 'wcpay/data';
+import {
+	FraudProtectionHelpText,
+	HighFraudProtectionModal,
+	StandardFraudProtectionModal,
+} from '../index';
+
+const ProtectionLevels = () => {
+	const [ isStandardModalOpen, setStandardModalOpen ] = useState( false );
+	const [ isHighModalOpen, setHighModalOpen ] = useState( false );
+	const { currencies } = useCurrencies();
+	const storeCurrency = currencies.default ? currencies.default : {};
+
+	const [
+		currentProtectionLevel,
+		updateProtectionLevel,
+	] = useCurrentProtectionLevel();
+
+	const handleLevelChange = ( level ) => {
+		updateProtectionLevel( level );
+	};
+
+	return (
+		<fieldset>
+			<ul>
+				<li>
+					<label htmlFor="fraud-protection__standard-level">
+						<div className="fraud-protection-single-radio-wrapper">
+							<input
+								name={ 'fraud-protection-level-select' }
+								id={
+									'fraud-protection-level-select_standard-level'
+								}
+								value={ 'standard' }
+								type={ 'radio' }
+								checked={
+									'standard' === currentProtectionLevel
+								}
+								onChange={ () =>
+									handleLevelChange( 'standard' )
+								}
+							/>
+							<p className="fraud-protection-single-radio-wrapper__item">
+								{ __( 'Standard', 'woocommerce-payments' ) }
+							</p>
+							<p
+								className={
+									'fraud-protection-single-radio-wrapper__item--recommended'
+								}
+							>
+								{ __(
+									'(Recommended)',
+									'woocommerce-payments'
+								) }
+							</p>
+							<HelpOutlineIcon
+								size={ 18 }
+								title="Standard level help icon"
+								onClick={ () => setStandardModalOpen( true ) }
+							/>
+							<StandardFraudProtectionModal
+								level="standard"
+								isStandardModalOpen={ isStandardModalOpen }
+								setStandardModalOpen={ setStandardModalOpen }
+								storeCurrency={ storeCurrency }
+							/>
+						</div>
+						<FraudProtectionHelpText level="standard" />
+					</label>
+				</li>
+				<li>
+					<label htmlFor="fraud-protection__high-level">
```

Comment:
The above comment also applies to this.

Reply:
Okay, fixed for real in [2d6aa3c](https://github.com/Automattic/woocommerce-payments/pull/5450/commits/2d6aa3cdc05974a29d189a513404bb22e8b0f4bd).